src/graphql/queries.js

```diff
@@ -216,15 +216,15 @@ subscription {
  * @type {string}
  */
 const WORKFLOWS_TABLE_QUERY = `
-  subscription {
-    workflows (ignoreInterval: 0) {
-      id
-      name
-      owner
-      host
-      port
-    }
+subscription WorkflowsTableQuery {
+  workflows (ignoreInterval: 0) {
+    id
+    name
+    owner
+    host
+    port
   }
+}
```

Comment:
Fixed indentation here, and added a "name", or "operation name" to each of our queries, so that it's easier to identify the queries in the `json-server` handler.